docset/winserver2025-ps/smbshare/New-SmbClientCertificateMapping.md

```diff
@@ -0,0 +1,322 @@
+    Subject = "CN=MyClientCertificate"
+    DisplayName = "MyClientCertificateMapping"
+}
+New-SmbClientCertificateMapping @params
+```
+
+This creates a new certificate mapping for an SMB client where a specific thumbprint, store name,
+issuer name, subject, and display name are specified for the certificate mapping.
+
+For example, if you have an SMB client named `MySMBClient` that needs to connect to a server using
+a specific certificate, you could use this cmdlet to create a certificate mapping that associates
+the client with the certificate.
+
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227967)
+or [Get-CimSession](/powershell/module/cimcmdlets/get-cimsession) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DisplayName
+
+Specifies a friendly name to display for the mapping.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Flags
+
+Specifies if `NamedPipes` are enabled for SMB over QUIC. The acceptable values for this parameter
+are:
+
+- `None`: Remove all flags
+- `AllowNamedPipe`: Enable use of named pipes in SMB over QUIC connections for this mapping (off by
+  default, overrides value of RestrictNamedPipeAccessOverQuic)
+- `DefaultCert`: Not used
+
+```yaml
+Type: Flags[]
+Parameter Sets: (All)
+Aliases:
+Accepted values: None, AllowNamedPipe, DefaultCert
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Force
+
+Forces the command to run without asking for user confirmation.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -IssuerName
+
+Filters the certificate mappings based on the issuer name of the certificate.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Namespace
+
+Specifies the namespace in which the certificate mappings are located. By default, the cmdlet
```

Comment:
specifies the namespace of the QUIC server. Example would be "server1.contoso.com"